documentation.html

```diff
@@ -278,7 +278,189 @@ <h1 class="heading--1-medium">Code</h1>
+						<h1 class="heading--1-medium">Variants</h1>
+						<h1 class="heading--3">
+							<ul class="feature__list">
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">avatar-img</code>
+									</span>
+								</li>
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">avatar-icon</code>
+									</span>
+								</li>
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">avatar-text</code>
+									</span>
+								</li>
+							</ul>
+						</h1>
+					</div>
+					<div class="main__example row-container">
+						<img
+							class="avatar-circle avatar-sm avatar-img"
+							src="./src/avatar.jpg"
+							alt=""
+						/>
+
+						<div class="avatar-circle avatar-text avatar-sm">AC</div>
+						<div class="avatar-circle avatar-sm avatar-icon">
+							<i class="fas fa-camera"></i>
+						</div>
+					</div>
+					<h1 class="heading--1-medium">Code</h1>
+					<div class="main__code">
+						<code class="main__code-bg">
+							<iframe
+								src="https://carbon.now.sh/embed/SaaQsFhl56ZvTxwTZNZb"
+								style="border: 0; transform: scale(1); overflow: hidden"
+								sandbox="allow-scripts allow-same-origin"
+							>
+							</iframe>
+						</code>
+						<!-- 	style="border: 0; transform: scale(1); overflow: hidden" -->
+					</div>
+					<div class="main__about">
+						<h1 class="heading--1-medium">Sizes</h1>
+						<h1 class="heading--3">
+							<ul class="feature__list">
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">avatar-circle</code>
+									</span>
+								</li>
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">avatar-box</code>
+									</span>
+								</li>
+							</ul>
+						</h1>
+					</div>
+					<div class="main__example row-container">
+						<div class="avatar-circle avatar-sm avatar-icon">
+							<i class="fas fa-camera"></i>
+						</div>
+						<div class="avatar-box avatar-sm avatar-icon">
+							<i class="fas fa-camera"></i>
+						</div>
+						<img
+							class="avatar-img avatar-box avatar-sm"
+							src="./src/avatar.jpg"
+							alt=""
+						/>
+						<img
+							class="avatar-img avatar-circle avatar-sm"
+							src="./src/avatar.jpg"
+							alt=""
+						/>
+						<div class="avatar-circle avatar-text avatar-sm">AC</div>
+
+						<div class="avatar-box avatar-text avatar-sm">AC</div>
+					</div>
+					<h1 class="heading--1-medium">Code</h1>
+					<div class="main__code">
+						<code class="main__code-bg">
+							<iframe
+								src="https://carbon.now.sh/embed/FR5aTIZsbVHVfDQmDALR"
+								style="border: 0; transform: scale(1); overflow: hidden"
+								sandbox="allow-scripts allow-same-origin"
+							>
+							</iframe>
+						</code>
+						<!-- 	style="border: 0; transform: scale(1); overflow: hidden" -->
```

Comment:
Remove unused code rather than commenting it

Reply:
Actually Its for my reference for the iframe!